application/templates/pages/guidance/index.html

```diff
@@ -12,7 +12,43 @@
 {% block content_primary %}
 
     <p>
-   Guidance for planning authorities adding data to the platform.
+      We’re working with 73 local planning authorities (LPAs) through the <a class="govuk-link" href="https://opendigitalplanning.org/community-members" target="_blank">Open Digital Planning community</a> to help them provide data to our platform. There are four steps to follow:
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>prepare data to the specifications</li>
```

Comment:
number these out as a list, 1 to 4

Reply:
Made some other changes to follow the guidance for steps: https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#bullet-points-steps